include/sys/ring_buffer.h

```diff
@@ -21,6 +21,11 @@ extern "C" {
 
 #define SIZE32_OF(x) (sizeof((x))/sizeof(uint32_t))
 
+/* Limit is used by algorithm for distinguishing between empty and full state.*/
```

Comment:
The limit is used by the algorithm for distinguishing between empty and full states.